Netkan/Sources/Curse/CurseMod.cs

```diff
@@ -34,7 +36,7 @@ public CurseFile Latest()
         /// <returns>The home</returns>
         public string GetProjectUrl()
         {
-            return "http://kerbal.curseforge.com/projects/" + ModId;
+            return "https://www.curseforge.com/project/" + id;
```

Comment:
This gives a 404, the old link works fine

Reply:
OK, will revert.